jbt-it/neomdb
# 169 client view project applications

How to test:
- go to http://localhost:3000/projekte/1/projektbewerbungen
- check if all details are displayed correctly, displaying 3 column could look bad depending on your screen size
- navigate to http://localhost:3000/projekte/2
- Click on "Alte Bewerbungen" and check if the applications are displayed correctly

closes #172
